commit e06cca765c19e1bc4b11cd2b0ca14d010c41d4a8 (refs/changes/50/4150/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-06-06T17:07:01-07:00 (1 year, 4 months ago)
    
    TRITON-476 amon build hardcodes python2.6, but only 2.7 is available on some build zone flavours

diff --git a/Makefile b/Makefile
index 1f25672..6de743a 100644
--- a/Makefile
+++ b/Makefile
@@ -65,11 +65,6 @@ NODE_DEV := ./node_modules/.bin/node-dev
 TAP := ./node_modules/.bin/tap
 JSSTYLE_FLAGS := -f tools/jsstyle.conf
 
-# Need to get our tools/bin on PATH to get our 'python'
-# first on the PATH. See RELENG-302.
-NPM := PATH=$(TOP)/tools/bin:$(TOP)/$(NODE_INSTALL)/bin:$(PATH) node $(TOP)/$(NODE_INSTALL)/bin/npm --tar=$(TAR)
-
-
 #
 # Repo-specific targets
 #
diff --git a/tools/bin/README.md b/tools/bin/README.md
deleted file mode 100644
index 3783346..0000000
--- a/tools/bin/README.md
+++ /dev/null
@@ -1,18 +0,0 @@
-<!--
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
--->
-
-<!--
-    Copyright (c) 2014, Joyent, Inc.
--->
-
-A bin dir for the build that requires (because of gyp in one of its deps) that
-the first `python` on the PATH be Python 2.6 or greater.  This is a pain with a
-smartos zone with both python24 and python26 installed, with `pkg_alternatives`
-installed /opt/local/bin/python might be python2.4 or python2.6 depending on
-install order.
-
-We already require "python2.6" to be on the PATH somewhere (but not first),
-so we use this bindir to reduce the former requirement to the latter.
diff --git a/tools/bin/python b/tools/bin/python
deleted file mode 100755
index 011a4e1..0000000
--- a/tools/bin/python
+++ /dev/null
@@ -1,12 +0,0 @@
-#!/bin/sh
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#
-
-#
-# Copyright (c) 2014, Joyent, Inc.
-#
-
-python2.6 "$@"
